src/devtools/client/debugger/src/utils/source.ts

```diff
@@ -398,7 +398,10 @@ export const getLineText = (asyncContent: SourceWithContent["content"], line: nu
   return lineText || "";
 };
 
-export function getTextAtPosition(asyncContent: SourceWithContent["content"], location: Location) {
+export function getTextAtPosition(
+  asyncContent: SourceWithContent["content"],
+  location: { column?: number; line?: number }
```

Comment:
Curious why this change? Doesn't seem like we changed any imports to e.g. remove the `Location` import so it seems unnecessary?

Reply:
I changed a couple of the types for functions that _called_ this one, and TS didn't like the mismatches. (Frankly, we have a plethora of variations on `Location`-type objects in this codebase, and there's a _lot_ of inconsistency in what fields and types get passed around).  So, narrowing this down to just the fields that it actually uses fixed a compilation error